.github/workflows/ci.yml

```diff
@@ -34,9 +34,13 @@ jobs:
           - "3.9"
           - "3.10"
           - "~3.11.0-0"
```

Comment:
@hynek note that with this, 3.11 is also allowed to fail. The conditional may need to be improved if you don't want that.